website/src/components/Blockquote.tsx

```diff
@@ -23,10 +22,10 @@ export const Blockquote: FunctionComponent<{
                 }
             >
                 {border && headline ? (
-                    <p className="font-weight-normal">
+                    <div className="case-studies__quote--in-content--section">
```

Comment:
In the replatform, we should make use of `blockquote`, `figcaption` and `cite` elements for more semantic markup instead of using divs.

Reply:
Sounds good, yeah. It would be nice to have a better and more consistent way of dealing with the border as well